pyteamcity.py

```diff
@@ -47,6 +47,12 @@ def _build_url(*args, **kwargs):
     return '/'.join(parts)
 
 
+def _catch_keyerror(y):
```

Comment:
You don't need a function for this.  You can just use `y.get('value')` where you are using this function instead